prime-dotnet-webapi/Controllers/HealthAuthoritiesController.cs

```diff
@@ -240,5 +240,62 @@ public async Task<ActionResult> UpdatePharmanetAdministratorContacts(int healthA
             await _healthAuthorityService.UpdateContactsAsync<HealthAuthorityPharmanetAdministrator>(healthAuthorityId, contacts);
             return NoContent();
         }
+
+        // HEAD: api/health-authorities/5/vendor/5
+        /// <summary>
+        /// Checks if the specified vendor is in use by a Health Authority site
+        /// </summary>
+        /// <param name="healthAuthorityId"></param>
+        /// <param name="vendorCode"></param>
+        [HttpHead("{healthAuthorityId}/vendor/{vendorCode}", Name = nameof(IsVendorInUse))]
+        [Authorize(Roles = Roles.EditSite)]
```

Comment:
Probably can be `Roles.ViewSite`

Reply:
Changed, but would that still allow for a deletion of a care type/vendor?